games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.Artsoft-rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
```

Comment:
I would say that if the game is already running (but got minimized or is not visible) then it would be nice to switch to it instead of launching a new instance.